HOWTORELEASE.rst

```diff
@@ -33,7 +36,7 @@ taking a lot of time to make a new one.
 
    Wait for the deploy to complete, then make sure it is `available on PyPI <https://pypi.org/project/pytest>`_.
 
-#. Merge the PR into ``master``.
+#. Merge the PR.
```

Comment:
```suggestion
#. Merge the PR.

#. If this is a maintenance release, cherry-pick the commit that updates the CHANGELOG into `master`, while keeping the `changelog` files intact so they get included in the next release as well.
```

Hmmm this is a little annoying I guess... 🤔

Reply:
I think I can put this in command form so it's a little easier 🤔

Reply:
Wouldn't/shouldn't changelog updates happen by merging maintenance into master?

Reply:
maintenance is intentionally divergent from `master` / `features`.  We actually want all of the changelogs that happen in 4.6 to additionally be part of the 5.x releases

So what we ~basically want to do is cherry pick the changes to `CHANGELOG.rst` and to `doc/en/announce`